packages/forma-36-website/src/components/Layout.js

```diff
@@ -43,67 +45,95 @@ const Layout = props => (
                 menuLinks {
                   name
                   link
-                  menuLinks {
-                    name
-                    link
-                  }
                 }
               }
             }
           }
         }
       }
-    `}
-    render={data => {
-      return (
-        <div css={styles.test}>
-          <Helmet
-            title={data.site.siteMetadata.title}
-            meta={[
-              {
-                name: 'description',
-                content: 'Forma 36 - The Contentful Design System',
-              },
-              {
-                name: 'keywords',
-                content: 'contentful, design, design-system',
-              },
-            ]}
-          >
-            <html lang="en" />
-          </Helmet>
-
-          {data.site.siteMetadata.promoText && (
-            <Promo
-              text={data.site.siteMetadata.promoText}
-              linkHref={data.site.siteMetadata.promoLink}
-              linkText={data.site.siteMetadata.promoLinkText}
-            />
-          )}
+      allFile(
+        filter: { sourceInstanceName: { eq: "forma-36-react-components" } }
+      ) {
+        edges {
+          node {
+            childMdx {
+              body
+              slug
+            }
+          }
+        }
+      }
+    }
+  `);
 
-          <Header />
+  const dataFromReadme = {};
 
-          <div css={styles.main}>
-            <Navigation
-              menuItems={
-                data.site.siteMetadata && data.site.siteMetadata.menuLinks
-              }
-              currentPath={props && props.location && props.location.pathname}
-            />
-            <Container
-              frontmatter={props.pageContext && props.pageContext.frontmatter}
-            >
-              {props.children}
-            </Container>
-          </div>
-        </div>
+  data.allFile.edges.map((item) => {
+    if (props.pageContext && props.pageContext.frontmatter.title) {
+      const arrFromSlug = item.node.childMdx.slug.split('/');
+      const trimmedTitle = props.pageContext.frontmatter.title.replace(
+        /\s/g,
+        '',
       );
-    }}
-  />
-);
+
+      if (arrFromSlug.indexOf(trimmedTitle) > -1) {
```

Comment:
Probably there is a more elegant way to do it?

Reply:
`Array.includes` is probably more readable.